### PR TITLE
Added step to provide password during (re)initialization

### PR DESCRIPTION
**Pull request by stuclem**
Added the step to provide the VIC appliance password when initializing and reinitializing the VIC appliance. Fixes https://github.com/vmware/vic-product/issues/2160.

@renmaosheng and @zjs can you please review or assign a reviewer?

Also, does this add a new option to the appliance initiation API? I just followed the instructions at https://vmware.github.io/vic-product/assets/files/html/1.4/vic_vsphere_admin/ova_reg_api.html and while they appeared to work (I saw the `operation complete 200` message), I do not believe that the appliance reinitialized. On the same subject, is there an option in the API to skip UI installation/upgrade? If there is, I missed that in 1.4.x.

### CODE REVIEW

**Comment by zjs** [comment]
> can you please review or assign a reviewer?

Adding @wjun, as he authored #2195.

> Also, does this add a new option to the appliance initiation API? ... On the same subject, is there an option in the API to skip UI installation/upgrade? If there is, I missed that in 1.4.x.

The UI calls the API, so the API should support anything the UI can do.

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 1]
```diff
@@ -108,6 +108,7 @@ If you need to deploy multiple appliances, you can use the initialization API to
 
     **IMPORTANT**: The installation process requires administrator credentials to register vSphere Integrated Containers Management Portal and Registry with the Platform Services Controller and to tag the appliance VM for use in Docker content trust. Administrator credentials are not stored on the appliance VM after use in the installation process. The vSphere Integrated Containers Management Portal and Registry services cannot start if you do not complete this step.
 
+12. Enter the root password for the vSphere Integrated Containers appliance, that you set when you deployed the OVA.
 12. To automatically install the vSphere Integrated Containers plug-in for vSphere Client, leave the **Install UI Plugin** check box selected, and click **Continue**.
```
Comment (wjun) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: The number is 13 here?

File: docs/user_doc/vic_vsphere_admin/reinitialize_appliance.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ You should reinitialize the appliance in the following circumstances:
 
     - If vCenter Server is managed by an embedded Platform Services Controller, leave the External PSC text boxes empty.
     - If the Platform Services Controller has changed since deployment, provide the new FQDN and administrator domain.
+4. Enter the root password for the vSphere Integrated Containers appliance, that you set when you deployed the OVA.
 4. To install or upgrade the vSphere Client plug-in, leave the **Install UI Plugun** checkbox selected, and click **Continue**.
```
Comment (wjun) [inline docs/user_doc/vic_vsphere_admin/reinitialize_appliance.md]: The number is 5 here?

Comment (wjun) [inline docs/user_doc/vic_vsphere_admin/reinitialize_appliance.md]: The number is 6 here?

**Pull request merged**